website/src/logged_in/prediction_tool/CreatePrediction.js

```diff
@@ -141,33 +145,12 @@ function CreatePrediction(props) {
                                                 onChange={(event) => setForecastedAttendance(event.target.value)}
                                             />
                                         </Grid>
+                                        
+                                        <FormControl fullWidth >
```

Comment:
<img width="1129" alt="Screen Shot 2021-04-09 at 9 43 41 AM" src="https://user-images.githubusercontent.com/6094957/114189431-31e09a00-9918-11eb-9d6e-b768307054f5.png">

1) Fix vertical spacing issue with right column. Both columns should start at the same place vertically on each page.
2) Add padding between specialized equipment and "Forecasted attendance"